install/ui/src/freeipa/hbactest.js

```diff
@@ -263,14 +263,30 @@ IPA.hbac.test_select_facet = function(spec) {
         that.table.multivalued = false;
 
         that.table.set_values = function(values) {
-            if (values && values.length && values[0] === '__external__') {
-                if (that.external_radio) that.external_radio.prop('checked', true);
-            } else {
-                that.table.table_set_values(values);
+            var records = that.table.records;
+            var has_values = values && values.length;
+
+            if (has_values && values[0] === '__external__') {
+                if (that.external_radio) {
+                    that.external_radio.prop('checked', true);
+                }
+                return;
             }
+
+            that.table.table_set_values(values);
         };
     };
 
+    that.show = function() {
+        that.facet_show();
+        that._make_sure_value_selected();
+    };
+
+    that.load_records = function(records) {
```

Comment:
These two methods, `show` and `load_records`, don't appear to be called explicitly. Are they used implicitly anywhere? Or are they just helpers to be used in the future??